index.html

```diff
@@ -72,5 +71,207 @@ <h3>LET’S CONNECT</h3>
+            2015
+          </span>
+        </p>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="card-buttons">
+          <li>
+            <button>Html</button>
+          </li>
+          <li>
+            <button>css</button>
+          </li>
+          <li>
+            <button>JavaScript</button>
+          </li>
+        </ul>
+        <a href="#" class="project-link">See project</a>
+      </div>
+    </section>
+    <section class="about-me">
+      <p class="title-about-me">
+        About Myself
+      </p>
+      <p class="text-about-me">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website. Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don’t hesitate to
+        contact me.
+      </p>
+      <h3>LET’S CONNECT</h3>
+      <ul class="ul-about-me">
+        <li class="icon">
+          <a href="https://github.com/">
+            <img src="./img/github.svg" alt="twitter logo" />
+          </a>
+        </li>
+        <li class="icon">
+          <a href="https://linkedin.com/">
+            <img src="./img/linkedin.svg" alt="linkedin logo" />
+          </a>
+        </li>
+        <li class="icon">
+          <a href="https://peace.com/">
+            <img src="./img/twofinger.svg" alt="twitter logo" />
+          </a>
+        </li>
+        <li class="icon">
+          <a href="https://twitter.com/">
+            <img src="./img/twitter.svg" alt="twitter logo" />
+          </a>
+        </li>
+        <li class="icon">
+          <a href="https://M.com/">
+            <img src="./img/M.svg" alt="M logo" />
+          </a>
+        </li>
+      </ul>
+      <button class="button-about-me">Get my resume</button>
+      <ul class="developer-languages-title">
+        <li>
+          <p class="subtitle-about-me">Languages</p>
+        </li>
+        <li>
+          <img src="./img/down.svg" alt="Right arrow" />
+        </li>
+      </ul>
+      <ul class="developer-languages">
+        <li class="items-developer-languages">
+          <img src="./img/js.svg" alt="" />
+          JavaScript
+        </li>
+        <li class="items-developer-languages">
+          <img src="./img/html.svg" alt="" />
+          HTML
+        </li>
+        <li class="items-developer-languages">
+          <img src="./img/css.svg" alt="" />
+          CSS
+        </li>
+      </ul>
+      <ul class="developer-languages-title">
+        <li><p class="subtitle-about-me">Frameworks</p></li>
+        <li><img src="./img/rigth.svg" alt="rigth_arrow" /></li>
+      </ul>
+      <div class="line"></div>
+      <ul class="developer-languages-title">
+        <li><p class="subtitle-about-me">Skills</p></li>
+        <li><img src="./img/rigth.svg" alt="rigth_arrow" /></li>
+      </ul>
```

Comment:
- [ ]  Kindly, wrappe all these element inside only one `ul` and make sure each of the direct child (`developer-languages-title` `developer-languages-title` `developer-languages-title` which represent Language, Framework and Skill) are wrapped in a `li`. Make sure to not change the  `developer-languages`, it's should remain a `ul`.

Your project structure should look like:
```
      <ul>
        <li class="developer-languages-title">
          <div>
            <h3 class="subtitle-about-me">Languages</h3>
            <img src="./img/down.svg" alt="Right arrow" />
          </div>
          <ul class="developer-languages">
            <li class="items-developer-languages">
              <img src="./img/js.svg" alt="" />
              JavaScript
            </li>
            <li class="items-developer-languages">
              <img src="./img/html.svg" alt="" />
              HTML
            </li>
            <li class="items-developer-languages">
              <img src="./img/css.svg" alt="" />
              CSS
            </li>
          </ul>
        </li>
        <li class="developer-languages-title">
          <p class="subtitle-about-me">Frameworks</p>
          <img src="./img/rigth.svg" alt="rigth_arrow" />
          <div class="line"></div>
        </li>
        <li class="developer-languages-title">
          <p class="subtitle-about-me">Skills</p>
          <img src="./img/rigth.svg" alt="rigth_arrow" />
          <div class="line"></div>
        </li>
      </ul>
```

Make sure to update the styling to match the figma design after the implementation. 

This will improve greatly the semantic of your webpage 😄 which will make your website much more accessible (for users, screen readers, and search engines).